bin/schema_generator.sh

```diff
@@ -141,6 +141,7 @@ function main() {
             --normalize-case \
             --force-nullable \
             --tuple-struct \
+            --allow-maps-without-value \
```

Comment:
requires https://github.com/mozilla/jsonschema-transpiler/pull/92

Reply:
This presumably requires a BigQuery sink change, too?

Reply:
yes